packages/lite-insight/src/attribution/dimensionDrillDownAttribution.ts

```diff
@@ -0,0 +1,89 @@
+import { locatedInInterval, enumerateAllDimensionCombinationsByDFS } from './util';
+import type { Datum } from '../interface';
+import type {
+  DataConfig,
+  CompareInterval,
+  InfoType,
+  FluctInfo,
+  AttributionResult,
+  TreeDim,
+  DataLocation,
+} from './types';
+
+/** Main function for dimension drill down attribution */
+const attribute2MultiDimensions: (
+  sourceData: Datum[],
+  dimensions: string[],
+  measure: string,
+  fluctuationDim: string,
+  baseInterval: CompareInterval,
+  currInterval: CompareInterval
+) => AttributionResult = (sourceData, dimensions, measure, fluctuationDim, baseInterval, currInterval) => {
+  /** remove invalid data */
+  const data = sourceData.filter((item) => !Object.values(item).some((v) => v === null || v === undefined));
+
+  const unitedInfo: InfoType = {
+    baseValue: 0,
+    currValue: 0,
+    diff: 0,
+  };
+
+  const resultTree: TreeDim = {};
+  const DictFlatten = {};
+  /** traverse the input data and build the result data structure; */
```

Comment:
use // to describe your code running logic, rather then /** */ (I am not sure)

Reply:
changed.